### PR TITLE
Patient Portal - My profile tab

### LINKED ISSUE #28: Patient Portal My Profile Page
**Is your feature request related to a problem? Please describe.**
Develop patient portal my profile page
Design: https://www.figma.com/file/mSVhvkZkuMD66AMNJCVIYE/MOH-NEHR-Project-Designs?type=design&node-id=121%3A2960&mode=design&t=W5cT3rAmjW8uUMJ4-1

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
MOH NEHR Profile Page

### PR DESCRIPTION
**Pull request by mrxnn**
resolves #28 

NEHR Client Application

### CODE REVIEW

File: nehr-patient-portal/modules/my-profile.tsx [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+                          className={cn(
+                            "pl-3 text-left font-normal",
+                            !field.value && "text-muted-foreground"
+                          )}>
+                          {field.value ? (
+                            format(field.value, "PPP")
+                          ) : (
+                            <span>{t("dateOfBirth")}</span>
+                          )}
+                          <CalendarIcon className="ml-auto h-4 w-4 opacity-50" />
```
Comment (janithRS) [inline nehr-patient-portal/modules/my-profile.tsx]: Please use react-icons/fa. `<FaCalendarAlt />`

File: nehr-patient-portal/modules/my-profile.tsx [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+                        </Button>
+                      </FormControl>
+                    </PopoverTrigger>
+                    <PopoverContent className="w-auto p-0" align="start">
+                      <Calendar
```
Comment (janithRS) [inline nehr-patient-portal/modules/my-profile.tsx]: AFAIK there is an issue with the shadcn calendar component: if you need to select a date from 1996 you need to go back month by month. I think it's OK to use the default input date picker for this. WDYT?

Comment (mrxnn) [reply nehr-patient-portal/modules/my-profile.tsx]: I didn't know this issue existed. I replaced it with the default input date picker.

File: nehr-patient-portal/modules/my-profile.tsx (hunk 1 above)
Comment (mrxnn) [reply nehr-patient-portal/modules/my-profile.tsx]: No longer using the calendar input

File: nehr-patient-portal/modules/my-profile.tsx [hunk 3]
```diff
@@ -1,7 +1,478 @@
...
+
+        <Button type="submit" className="mt-8">
+          Submit
+        </Button>
+      </form>
+    </Form>
+  );
+};
+
+const Heading = ({ children }: { children: any }) => {
```
Comment (janithRS) [inline nehr-patient-portal/modules/my-profile.tsx]: Shall we this to components?

File: nehr-patient-portal/tailwind.config.js [hunk 4]
```diff
@@ -74,4 +74,4 @@ module.exports = {
     },
   },
   plugins: [require("tailwindcss-animate")],
-}
```
Comment (janithRS) [inline nehr-patient-portal/tailwind.config.js]: EOF

File: nehr-patient-portal/modules/my-profile.tsx (hunk 3 above)
Comment (janithRS) [inline nehr-patient-portal/modules/my-profile.tsx]: Also, it's better to have a defined prop interface in case we need to expand the header props later.

**Review by janithRS — CHANGES_REQUESTED** [review]

File: nehr-patient-portal/modules/my-profile.tsx (hunk 3 above)
Comment (mrxnn) [reply nehr-patient-portal/modules/my-profile.tsx]: Moved

**Review by janithRS — APPROVED** [review]

**Pull request closed without merging**